google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryOptions.java

```diff
@@ -38,6 +38,7 @@ public class BigQueryOptions extends ServiceOptions<BigQuery, BigQueryOptions> {
   private final String location;
```

Comment:
It's a good idea to increment the serialVersionUID.

Reply:
Sounds good, is it just incremented by 1 each time? And in this case since the UID is negative, should it still be incremented or decremented?

Reply:
I just increment by 1 each time.
For negative, I continue to increment. I.e., -11->-10.

Reply:
Resolved.